tests/ert_tests/ert3/engine/integration/test_engine.py

```diff
@@ -189,9 +189,12 @@ def test_run_once_polynomial_evaluation(
         ert3.engine.run(experiment_run_config, workspace, "evaluation")
     with assert_clean_workspace(workspace):
         with pytest.raises(
-            ValueError, match="Experiment evaluation has been carried out"
-        ):
+            ert.exceptions.ExperimentError,
+            match="Experiment 'evaluation' has been carried out already.",
+        ) as exc:
             ert3.engine.run(experiment_run_config, workspace, "evaluation")
+            assert exc.experiment_name == "evaluation"
+            assert exc.suggest_clean
```

Comment:
Are these still valid?

Reply:
huh, evidently those do not get executed within a `pytest.raises`. Basically dead code that I forgot...